packages/object-lists/migrations/0000_bored_silver_surfer.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE `object_list_item` (
+	`id` serial AUTO_INCREMENT NOT NULL,
```

Comment:
Is there a way to use spaces instead of tabs (for consistency with the rest of the codebase)?